### PR TITLE
Config lock bot

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Review by techknowlogick — APPROVED** [review]

File: .github/lock.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+exemptLabels: []
+
+# Label to add before locking, such as `outdated`. `false` is disabled
+lockLabel: false
+
+# Comment to post before locking.
+lockComment: >
+  This thread has been automatically locked since there has not been
+  any recent activity after it was closed. Please open a new issue for
+  related bugs.
```
Comment (6543) [inline .github/lock.yml]:
```suggestion
  related bugs and link to relevant comments in this thread.
```

Comment (6543) [reply .github/lock.yml]: @silverwind, @techknowlogick :point_up:  ->  :+1: / :-1: ?

Comment (6543) [reply .github/lock.yml:20]: @silverwind

**Review by jolheiser — APPROVED** [review]
Assuming this only acts on closed issues/PRs LGTM

**Pull request merged**